1/p와-y개수-세기.js

```diff
@@ -0,0 +1,22 @@
+// p와 y개수 세기
+
+function solution(s) {
+  let answer = true;
+  let pNum = 0; // p 개수 저장 변수
+  let yNum = 0; // y 개수 저장 변수
```

Comment:
```suggestion
  let pCnt = 0; // p 개수 저장 변수
  let yCnt = 0; // y 개수 저장 변수
```
보통 카운팅 하는 변수를 선언할 때는 cnt를 많이 이용합니다.